tests/test_table.py

```diff
@@ -109,6 +109,24 @@ def test_field_ranges(self, full_table):
 
 
 class TestTableCRUD:
+    http_mocks = mock_google_sheets_responses([
```

Comment:
This is already in the fixture called `table`.

If you need access to other variables lie `sa, table_range, http_mocks` I recommend puting them on their on fixtures.

Reply:
I don't know if this comment was meant to be on this line but I don't understand it at all xd

Reply:
Yes, it's for lines 112-129. I removed all those fixtures (http_mock, SpreadsheetApp, table_range and table) from this file and put them on `conftest.py`

I don't see the need for these here.

Reply:
Got it! I fucked up when merging the master branch to my dev branch and reverted your change of removing these lines from there xD

Reply:
That conftest.py file is amazing. It simplifies the tests so much.